### ISSUE TITLE
Custom error pages for websites

### ISSUE THREAD

**Issue by duijf** [svsticky/sadserver#207]
@RCdeWit asked if we can allow custom error pages for individual sites. He needs it for exec(ut). He only needs a 404 template at the moment, but we might want to provide a general solution as well. 

From a cursory glance, we don't seem to support this yet. 

On mobile, so sorry if this is a duplicate

**Comment by maartenberg** [comment]
This should be doable using an include containing something like this:

```
error_page 404 /execut_404.html;
location = /execut_404.html {
	root /var/www/symposium/custom-errors;
	internal;
}
```
([Adapted from](https://www.digitalocean.com/community/tutorials/how-to-configure-nginx-to-use-custom-error-pages-on-ubuntu-14-04))

And then adding that include to a site's `extra_includes` (and ensuring the file exists).

**Comment by tomwassenberg** [comment]
@maartenberg Looks good, being reusable for all the execut websites. Maybe we can even extend it to be usable for all websites?

For that we'd probably need to add the snippet to the Ansible template ([website.conf.j2](https://github.com/svsticky/sadserver/blob/master/ansible/templates/etc/nginx/sites-available/website.conf.j2)), e.g. like this:

```
{% if item.custom_404_page %}
error_page 404 /404.html;
location = /404.html {
	root /var/www/{{ item.user }}/custom-errors;
	internal;
}
{% endif %}
```

Ensuring the file exists is not even needed I think, since you probably get a 404 either way.

**Comment by maartenberg** [comment]
Fixed with f5bab023d67035ee7b0c82a30e05293d48640214
